src/Microsoft.AspNet.Mvc.Core/ViewComponents/ViewViewComponentResult.cs

```diff
@@ -92,12 +92,20 @@ public void Execute([NotNull] ViewComponentContext context)
 
             var view = FindView(context.ViewContext, viewEngine, qualifiedViewName);
 
+            var htmlHelperOptions = new HtmlHelperOptions
+            {
+                ClientValidationEnabled = context.ViewContext.ClientValidationEnabled,
+                Html5DateRenderingMode = context.ViewContext.Html5DateRenderingMode,
+                ValidationMessageElement = context.ViewContext.ValidationMessageElement,
+                ValidationSummaryMessageElement = context.ViewContext.ValidationSummaryMessageElement
+            };
```

Comment:
View components are _never_ created at the top level.  The `ViewComponentContext` contains the `ViewContext` for the next-outer scope.  So the updated code below is another case where the `TempData` argument was added in error.  Should be using the `ViewContext` copy constructor.